lib/npm-apm.js

```diff
@@ -45,6 +45,52 @@ function provideBuilder() {
           sh: false
         });
       }
+
+      function makeConfig(tool) {
+        var _args = [];
+        for (var i = 1; i<arguments.length; i++) {
+          _args.push(arguments[i]);
+        }
```

Comment:
You can use ES6 here (via [babe](https://babeljs.io/) and declare `_args` with `let` to get locally scoped variables.